install_config/registry/extended_registry_configuration.adoc

```diff
@@ -581,6 +586,35 @@ be safe to enable its support in the registry. See the
 xref:docker-registry-configuration-reference-middleware[middleware
 configuration reference] above for particular option.
 
+[[docker-registry-configuration-reference-openshift]]
+=== OpenShift
+
+This section allows to configure global settings related to {product-title}
```

Comment:
s/This section allows to/This section reviews the configuration of global settings